### ISSUE TITLE
Correct type of policy_id  in email integration

### ISSUE THREAD

**Issue by mkochco** [BetterStackHQ/terraform-provider-better-uptime#52]
Looks like there was a recent issue with [incoming webhook integrations](https://github.com/BetterStackHQ/terraform-provider-better-uptime/pull/47).   I'm trying to import an existing email integration and receiving 

```
Error: json: cannot unmarshal string into Go struct field emailIntegration.data.attributes.policy_id of type int
```

**Comment by adikus** [comment]
Thanks @mkochco 
I'll update the Email integrations same way as https://github.com/BetterStackHQ/terraform-provider-better-uptime/pull/47 I should have realized that both need to be fixed.

**Comment by mkochco** [comment]
Coming on 7 weeks.  Any progress here?  Any workarounds?  I'm currently blocked trying to create a new email integration

```
 Error: json: cannot unmarshal string into Go struct field emailIntegration.data.attributes.policy_id of type int
│ 
│   with module.integrations.betteruptime_email_integration.solarwinds_events,
│   on modules/integrations/main.tf line 1, in resource "betteruptime_email_integration" "solarwinds_events":
│    1: resource "betteruptime_email_integration" "solarwinds_events" {

```

**Comment by gyfis** [comment]
@mkochco Hi Mark 🙌 this should be available in the latest release (0.3.20)
